_posts/2017-08-18-forbes.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Forbes
+sourceURL: https://digiday.com/media/new-mobile-site-forbes-boosted-impressions-per-session-10-percent/
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Offline
+- Service Worker
+---
+
+**Forbes’** PWA loads in **2.5s** on mobile compared to 6.5 seconds for its previous site. Impressions per visit are up **10%**.
```

Comment:
Please be consistent in either abbreviating or not abbreviating the word "seconds."